pkg/customflag/model.go

```diff
@@ -52,7 +53,7 @@ type clusterConfigFlag struct {
 
 type destroyFlag bool
 
-type TestCaseFlag func(deployWorkload bool)
+type TestCaseFlag func(applyWorkload, deleteWorkload bool)
```

Comment:
since we are refactoring this , can  we add a type that is more flexible and generic ? so we can also add the test cases that does not apply nor even delete workloads.

Reply:
I need to think on this. My gut is telling me that making this more generic would actually do the opposite and cause it to be over-complicated and hacky since we'd need to be able to accept functions with any signature in that case and then be able to pass the args via some flag.

Reply:
agree
we need to think which will happens more
 
the majority of testcases now and in the future will require some workload right ?

rn i only see the selinux ones, not having